### PR TITLE
Console component

### PR DESCRIPTION
**Pull request by monirakhan**

### CODE REVIEW

File: src/components/Console.vue [hunk 1]
```diff
@@ -0,0 +1 @@
+//Define new console component
```
Comment (PC98) [inline src/components/Console.vue]: This is a good start, but we need some code in here!

File: src/App.vue [hunk 2]
```diff
@@ -4,14 +4,17 @@
     <HelloWorld msg="Welcome to Your Vue.js App"/>
   </div>
 </template>
+<Console>
```
Comment (PC98) [inline src/App.vue]: This needs to move inside the `<div>` in the `<template>`

**Review by PC98 — CHANGES_REQUESTED** [review]

**Comment by PC98** [comment]
@monirakhan I would recommend pulling from monirakhan:console_component and then addressing the requested changes.

File: src/components/Console.vue (hunk 1 above)
Comment (monirakhan) [reply src/components/Console.vue]: Gotcha!

**Comment by PC98** [comment]
You have all the code right. But we need to move it to different places. You should move the HTML and CSS for the Console component to its .vue file. So `src/components/Console.vue` should look something like:
```html
<template>
  <div id="console">
    <div style="height: 48px;"></div>
  </div>
</template>

<style>
#console {
  position: absolute;
  width: 286px;
  margin: 10px;
  padding: 8px 8px;
  background-color: rgb(225, 225, 225);
  box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25), 0 10px 10px rgba(0, 0, 0, 0.22);
  border-radius: 3px;
  z-index: 0;
}
</style>
```
Then you can remove the HTML you added to `public/index.html` and the CSS you added to `src/App.vue`

**Pull request closed without merging**